src/main/java/com/mojang/brigadier/CommandDispatcher.java

```diff
@@ -435,8 +436,8 @@ public int execute(final ParseResults<S> parse) throws CommandSyntaxException {
      * listed as multiple entries: the parent node, and the child nodes.
      * For example, a required literal "foo" followed by an optional param "int" will be two nodes:</p>
      * <ul>
-     *     <li>{@code foo}</li>
```

Comment:
Why did you unindent HTML elements?

Reply:
Sigh. That was IntelliJs autoformatter. I surrounded the ones I wrote with formatter off tags to prevent that, but it seems I missed some. Thanks, I will restore that :)